sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md

```diff
@@ -394,6 +394,9 @@ The valid values are:
 - WebPreviewableFiles
 - ServerRenderedFilesOnly
 
+> [!NOTE]
```

Comment:
[!NOTE] should not be used on cmdlet's articles as it can't be rendered in Get-Help output. Please replace with regular note.